packages/graphback-cli/src/templates/configTemplates.ts

```diff
@@ -91,15 +94,18 @@ export const createConfig = async (database: string, client: boolean) => {
             "format": "ts",
             "outputPath": "./src/resolvers"
           }
-        },
-      },
-      dbmigrations: {
-        "connection": JSON.parse(dbConfig),
-        "client": database
+        }
       }
     }
   };
 
+  if(database === "pg") {
+    graphqlConfig.extensions.graphback.dbmigrations = {
+      "connection": JSON.parse(dbConfig),
+      "client": database
```

Comment:
I am adding it back here for pg. 
If database is pg it is supposed to get added back. 
Will it not work?